templates/zerver/help/make-a-user-an-administrator.md

```diff
@@ -0,0 +1,29 @@
+# Make a user an administrator
+
+> With great power comes great responsibility.
+
+By default, everyone in a realm is a user which limits them from modifying realm wide settings, such as the organization name, activating/deactivating users, deleting streams, etc. Zulip realm administrators can give a user administrative rights. To do so, follow the steps below:
+
+1. Click the (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
+
+2. Select **Administration** from the dropdown menu.
+
+    ![admin](/static/images/help/admin.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the **Administration** page. Click on **Users** in the tabs list.
+
+    ![admin panel](/static/images/help/admin-panel.png)
+
+4. In the **Users** section, you will be presented with various control options over your users. Click on **Make admin** for the user(s) that you wanted to make an administrator.
+
+    ![make admin](/static/images/help/make-admin.png)
+
+After clicking the **Make admin** button, the selected user's browser would reload and gain administrative access immediately.
```

Comment:
and the user will gain administrative privileges (the browser isn't the one gaining admin access lol)